### PR TITLE
ccus mod updates

### PR DESCRIPTION
**Pull request by ez7532**

### CODE REVIEW

File: src/types/modifications/CCUS.jl [hunk 1]
```diff
@@ -3,12 +3,30 @@
...
+* `<name>_ccus_paths` - contains all the pathways to selling CO₂. The table includes the producing region, the sequstering region and the type of storage, market steps, the quantity available for storage, and the costs of storage and transport.
+
+### Table Columns Added
+* `(:ccus_paths, :path_idx)` - the index of this path. 
+* `(:ccus_paths, :price_total)` - the cost of transporting and storing a short ton of CO₂ in this storage pathway.
+* `(:ccus_paths, :stored_co2)` - the amount of CO₂ stored via this storage path, measured in short tons.
+* `(:ccus_paths, :storer_cost)` - total storage cost of CCUS via this pathway, from the perspective of the sequesterer.
+* `(:ccus_paths, :storer_revenue)` - total revenue earned from storage for CCUS via this pathway.  This is the amount paid by the EGU's to the sequesterer, equal to the clearing price minus the transport cost.
+* `(:ccus_paths, :storer_profit)` -  total profit earned by storing carbon via this pathway, equal to the revenue minus the cost.
+* `(:gen, :capt_co2)` - the rate of capture of CO2 (calculated from emis_co2 and capt_co2_percent).
```
Comment (Ethan-Russell) [inline src/types/modifications/CCUS.jl:24]: Minor thing here.  In the julia REPL, looks like markdown is mis-interpreting your underscores in emis_co2 and capt_co2_percent as you trying to underline.  For example `_this_` is interpreted as _this_.  To correct this, you can surround the column names that contain underscores with a tick mark (i.e. ` )

File: src/types/modifications/CCUS.jl [hunk 2]
```diff
@@ -3,12 +3,30 @@
...
+This is a [`Modification`](@ref) that sets up markets for carbon captured by generators.
+
+### Keyword Arguments    
 * `file` - a file to the table containing markets/prices for buying/selling carbon dioxide.  See the `ccus_paths` table below, or [`summarize_table(::Val{:ccus_paths})`](@ref)
 * `groupby` - a `String` indicating how markets are grouped.  I.e. "state".
 * `co2_scalar` - a `Float64` for how much to scale the co2 variables by.  This helps with numerical instability, given that some CO₂ steps can be very large and could bloat the RHS and bounds range of the model.  A good rule of thumb is that this should be no less than `1e4` times smaller than `co2_step_quantity_limit`.
 * `co2_step_quantity_limit` - A `Float64` for the maximum quantity of CO₂ that can be stored in any step.
 
+### Tables Added
+* `<name>_ccus_paths` - contains all the pathways to selling CO₂. The table includes the producing region, the sequstering region and the type of storage, market steps, the quantity available for storage, and the costs of storage and transport.
```
Comment (Ethan-Russell) [inline src/types/modifications/CCUS.jl:15]: I believe the table being created is simply `ccus_paths` (i.e. no name in front of it).

File: src/types/modifications/CCUS.jl [hunk 3]
```diff
@@ -31,19 +49,28 @@ Creates the following tables in `data`:
...
-
-Creates the following constraints
-* `cons_co2_stor[1:nrow(ccus_storers), 1:nyear]` - the CO₂ stored at each injection site must not exceed `step_quantity`
-* `cons_co2_bal[1:nrow(ccus_producers), 1:nyear]` - the CO₂ balancing equation for each region, i.e. `co2_prod == co2_sent`.
-
-## Accessing Results
-Results are stored in 2 places; the `ccus_paths` table, and the `gen` table.
+### Model Modification: 
+* Variables
+    * `co2_trans[1:nrow(ccus_paths), 1:nyear]` - the amount of captured CO₂ transported to each step along the producer-producer pathway. It is measured in MMT and bounded by [0, max_co2_storage * 1.1].
```
Comment (Ethan-Russell) [inline src/types/modifications/CCUS.jl:54]: Surround `max_co2_storage` with ticks

File: src/types/modifications/ReserveRequirement.jl [hunk 4]
```diff
@@ -47,7 +47,18 @@ Representation of reserve requirement, such that the sum of eligible power injec
   * `cons_pres_<name>` - (nsubarea x nyr x nhr) Constrain that `pres_total_subarea_<name> ≥ pres_req_subarea_<name>`.
 
 ### Results Formulas:
+* `(:<name>_requirements, :pres_req)` - the hourly required reserve power for the region.
+* `(:<name>_requirements, :pres_supply)` - the hourly supplied reserve power for the region. 
+* `(:<name>_requirements, :eres_supply)` - the supply for reserve energy. May be misleading but useful for calculations. 
+* `(:<name>_requirements, :cost_total)` - cost paid for the supplied reserve power. 
+* `(:<name>_requirements, :price_avg)` - the total cost divided by the supply of reserve power. This represents the shadow price on the capacity constraint, in dollars per MW reserve capacity supplied per hour. 
+* `(:<name>_requirements, :pres_flow)` - the hourly power flowing out of the region. May be misleading when aggregating above the subarea level. 
+* `(:bus, :<name>_cost_result)` - the total rebate paid by users to EGUs (and storage if present) from the $(mod.name) reserve requirement, not including merchandising surplus.
```
Comment (Ethan-Russell) [inline src/types/modifications/ReserveRequirement.jl:56]: The dollar sign inside of a string in julia represents string interpolation.  For example:
```julia
julia> a = 5
5
julia> s = "this is a string and here is a: $(a)"
"this is a string and here is a: 5"
```

You'll need to remove all the references to `$(mod.name)` from the docstring.  You can probably just remove them or replace with "the reserve requirement" or similar.

**Review by Ethan-Russell — CHANGES_REQUESTED** [review]
Overall looking really good so far!  Just a couple of comments to address.

File: src/types/modifications/CCUS.jl (hunk 1 above)
Comment (ez7532) [reply src/types/modifications/CCUS.jl]: Hi Ethan, thanks for the review! I made all the amendments and re-committed and seems like everything is in order. thanks!

**Review by Ethan-Russell — APPROVED** [review]
Looks great!  I'm approving the PR.

**Pull request merged**